lib/fastlane/plugin/fueled/actions/ci_checks_flutter.rb

```diff
@@ -26,6 +28,20 @@ def self.description
         "Run tests and check code coverage"
       end
 
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(
+            key: :skip_integration_tests,
+            env_name: "SKIP_INTEGRATION_TESTS",
+            description: "If true, only unit tests are executed",
```

Comment:
We may add UI testing in the future? Maybe:

If true, the integration tests will be skipped.

Reply:
Yeah makes sense, thanks Hadi! 🙌